go.mod

```diff
@@ -3,19 +3,19 @@ module sigs.k8s.io/controller-runtime
 go 1.16
 
 require (
-	github.com/evanphx/json-patch v4.9.0+incompatible
+	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-logr/logr v0.4.0
 	github.com/go-logr/zapr v0.4.0
 	github.com/googleapis/gnostic v0.5.5 // indirect
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/imdario/mergo v0.3.12 // indirect
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.12.0
-	github.com/prometheus/client_golang v1.10.0
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
```

Comment:
Do we need the new minor for gomega?

Reply:
Nothing exciting here. CAPA is using some of the deprecated pieces, but doesn't look like CAPI is.